cmd/sink/routers/result.go

```diff
@@ -8,12 +8,8 @@ import (
 	"github.com/cloudevents/sdk-go/v2/protocol"
 )
 
-func NewCEResult(statusCode int, err error) protocol.Result {
-	if statusCode >= 200 && statusCode < 400 {
-		// Cloud Event Processed successfully
-		return cloudevents.NewHTTPResult(statusCode, "cloud event processed successfully")
-	} else if err == nil {
-		return cloudevents.NewHTTPResult(statusCode, "an error occurred while processing the cloud event")
-	}
-	return cloudevents.NewHTTPResult(statusCode, "an error occurred while processing the cloud event", err.Error())
+// knative eventing expects that a response to a cloud event has an empty body, so NewHTTPResult should be called with
+// an empty string.
+func NewCEResult(statusCode int) protocol.Result {
+	return cloudevents.NewHTTPResult(statusCode, "")
```

Comment:
This method seems like a waste now. Just call cloudevents.NewHTTPResult directly.

Reply:
Good point.  Then can you change the comment from "should be" to "must be"?

Reply:
done